tests/Feature/ComponentSecurityTest.php

```
@@ -92,42 +92,42 @@ public function it_sanitizes_user_arguments_before_delegation()
+
         $component = [
             'name' => 'test',
             'binary' => $testBinary,
         ];
 
         // Capture output
         ob_start();
-        
+
         // Delegate with dangerous arguments
         $exitCode = $delegationService->delegate(
```

PHPMD (2.15.0)

Avoid unused local variables such as '$exitCode'. (Unused Code Rules)

(UnusedLocalVariable)